doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-test-scenario-export-import-spreadsheet-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-designer-test-scenario-export-import-spreadsheet-con']
+= Exporting and Importing a test scenario spreadsheet
+
+This sections will help you understand how to export and import a spreadsheet of test scenario using the Test Scenario designer. The export and import functions of the Test Scenario designer currently supports the `.csv` file format. These spreadsheets can be managed by software such as Microsoft Excel or LibreOffice Calc. This will help you to analyze and maintain the spreadsheet data.
```

Comment:
Should we avoid the wording "help you understand"?

s/This sections/These sections

s/import a spreadsheet of test scenario/import a test scenario spreadsheet/import a spreadsheet for test scenario

s/These spreadsheets can be managed by/You can manage the spreadsheets by using

I wonder if this can be shortened and simplified, maybe something like below?

"These sections show how to export and import test scenario spreadsheets in the Test Scenario designer. Test Scenario designer supports the `.csv` file format. You can analyse and manage test scenario spreadsheets with software sch as Microsoft Excel or LibreOffice Calc."

Reply:
Done

Reply:
I still have the question about setting context. Why and when would a user need to export or import test scenario data? This (IMHO) is a key piece of information given that we are creating modular content. Why do I (as the user) care about this topic?